app/appView.js

```diff
@@ -62,7 +62,9 @@ view.singlePage = function(){
   });
 }
```

Comment:
This singlePage function displays both the About and Chickens tabs when the page loads. This makes it odd when you click on the nav items and then one or the other disappears. Shouldn't the page load at the same given starting point? Also, this code looks like it could be refactored to be clearer/cleaner...